packages/client/src/Call.ts

```diff
@@ -1435,10 +1435,11 @@ export class Call {
   /**
    * Starts the livestreaming of the call.
    */
-  goLive = async () => {
+  goLive = async (notify?: boolean) => {
```

Comment:
```suggestion
  goLive = async ({ notify = false }: { notify?: boolean }) => {
```

I think for consistency (see `call.get()`) and future-proofness, this method should accept an object with params.

Reply:
make sense